### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TiagoF99**

### CODE REVIEW

**Comment by GintasS** [comment]
@Zmwang622 should we include the link to new grads repository?

**Comment by Zmwang622** [comment]
Yeah

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ And we're back 😤🎉! Use this repo to share and keep track of any tech-relat
 
 🤗 **Contribute by submitting a [pull request](https://github.com/susam/gitpr#create-pull-request) or [filling out this form](https://bit.ly/3d5O76c)!**  🤗
 
+[New Grad 2022 Listing](https://github.com/coderQuad/New-Grad-Positions-2022)
```
Comment (Zmwang622) [inline README.md:8]: Can we format this a bit better? Here's what we did [last year](https://github.com/pittcsc/Summer2022-Internships/tree/07d3871c0e021ca414ec8f2c94d3563bc06d386d)

**Review by Zmwang622 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GintasS) [reply README.md]: Fixed.

**Comment by GintasS** [comment]
@Zmwang622 Even though I have merged this, is the link https://github.com/coderQuad/New-Grad-Positions-2022 correct? This is a 3rd party organization, the repo is not from Pitt?

**Comment by Zmwang622** [comment]
Pitt didn't start one this year (I'm too lazy). This one is a fork of ours.

**Pull request merged**